SDWebImage/Core/SDImageIOAnimatedCoder.m

```diff
@@ -301,12 +301,12 @@ - (NSData *)encodedDataWithImage:(UIImage *)image format:(SDImageFormat)format o
     }
     
     NSMutableData *imageData = [NSMutableData data];
-    NSString *imageUTType = self.class.imageUTType;
```

Comment:
This should use the input argument `format`, but not `self.class.imageUTType`. Because user may call this manually